src/Service/ImageManipulationService.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Service;
+
+use Intervention\Image\ImageManager;
+use League\Flysystem\FilesystemOperator;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class ImageManipulationService
+{
+    public const IMAGE_MIME_TYPES = [
+        'image/jpeg',
+        'image/png',
+        'image/gif',
+    ];
+
+    private const DEFAULT_SIZE_RESIZE = 1000;
+    private const DEFAULT_SIZE_THUMB = 400;
+
+    private $imageManager;
+    private $tmp = true;
```

Comment:
Je trouve cette propriété pas très explique dans le contexte de cette classe. A quoi elle sert @numew ?

Reply:
Est ce pour déterminer si on utilise la ressource temporaire ou non ?

Reply:
Oui c'est pour déterminer si on travaille sur une image dans le repertoire tmp ou à la racine du bucket

Reply:
Du coup useTmp ça sera plus clair, qu'en penses tu ?

Reply:
tant qu'a être plus explicite je mettrais bien "isInTmpDir" ?

Reply:
Oui il est bien d'ajouter le `Dir` 
Apres dans ce contexte la je préfère un d'action qu'un verbe d'état, ça évite le `in`
useTmpDir que le `isInTmpDir` On peut demander l'avis de @emilschn et @hmeneuvrier

Reply:
ok